### ISSUE TITLE
How to batch insert into table with pg

### ISSUE THREAD

**Issue by xfliudx** [brianc/node-postgres-docs#54]
When I batch insert multiple rows, how to write parameterized sql?   
Thanks a lot

**Comment by wbt** [comment]
I agree, a well-documented performant example similar to "[Parameterized query](https://node-postgres.com/features/queries#Parameterized%20query)" showing multiple rows would be quite helpful.
